api/api_controller_test.go

```diff
@@ -37,19 +35,20 @@ import (
 	"github.com/treeverse/lakefs/upload"
 )
 
+const timeout = 10 * time.Second
```

Comment:
>     1. Is it possible to set it to 30 sec - like the default

Sure, but why?  This is a test.  Every query should finish really quickly.

>     2. Question - you used the NewXXXParamsWithTimeout api version because you wanted to control the default? we can just use the NewXXXParams version.

The default is too long for a test.  If something is wrong I want to get as many errors (in-test timeouts) as possible.  And there's never a reason to wait.

Reply:
1. no problem - my thought was about less code
2. np II